hoodie-client/src/main/java/com/uber/hoodie/table/HoodieCopyOnWriteTable.java

```diff
@@ -625,13 +548,13 @@ public String toString() {
   class BucketInfo implements Serializable {
 
     BucketType bucketType;
-    String fileLoc;
+    String fileIdHint;
```

Comment:
why is this fileIdHint ? Should this be fileLocHint ? Or even fileIdPrefix ?

Reply:
Renamed to fileIdPrefix